CHANGELOG.md

```diff
@@ -5,6 +5,25 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v3.63.0](https://github.com/buildkite/agent/tree/v3.63.0) (2024-02-14)
+[Full Changelog](https://github.com/buildkite/agent/compare/v3.62.0...v3.63.0)
+
+### Fixed
+- Fix pipeline interpolation case sensitivity on Windows, and runtime environment variable precedence [#2624](https://github.com/buildkite/agent/pull/2624) (@triarius)
```

Comment:
Maybe we should replicate the examples from the `go-pipeline` changelog: https://github.com/buildkite/go-pipeline/releases/tag/v0.4.0

Both changes look like likely to be subject to Hyrum's Law.

Reply:
I've largely copied from the go-pipeline release notes, with some alterations: removing code-level specifics and changing "runtime env" to "agent environment" to be consistent with the linked docs. LMK what you think